resources/integration_test/tests/dbmd_empty_result_sets.yml

```diff
@@ -5,6 +5,7 @@ tests:
     meta_function: [getProcedures, integration_test, null, '%']
     expected_result: []
     row_count: 0
+    ordered: true
```

Comment:
nit: Up to you whether to remove, these tests return 0 rows and the `ordered` field would have no effect and wouldn't be needed.

Reply:
Yes, I know, but I want to keep the marker, so that the day we add procedures support, even if it is very unlikely we'll ever do, then things are "ready"